src/config/resharding.rst

```diff
@@ -32,7 +32,17 @@ Resharding Configuration
         will start to fail with the ``max_jobs_exceeded`` error::
 
              [reshard]
-             max_jobs = 25
+             max_jobs = 48
+
+    .. config:option:: max_history
+
+        The max length of each jobs event log. The history records job starts,
+        stop, retries etc. There is probably not much reason to change this
+        unless use specifies larger number of ``max_retries``::
```

Comment:
I'm not sure what is meant here.  Maybe this?

```suggestion
        unless you specify a larger number of ``max_retries``::
```

Reply:
I think I'll remove that sentence. It doesn't make much sense in general.